manifest.md

```diff
@@ -22,7 +22,9 @@ A client will distinguish a manifest list from an image manifest based on the Co
 - **`mediaType`** *string*
 
   This REQUIRED property contains the MIME type of the manifest list.
-  For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
+  For this version of the specification, this MUST be set to
```

Comment:
The rest of the file is not wrapped but instead one-sentence-per-line (or one-paragraph-per-line), could you do similarly here?